pkg/dependency/parser/java/pom/parse_test.go

```diff
@@ -1630,6 +1630,21 @@ func TestPom_Parse(t *testing.T) {
 				},
 			},
 		},
+		{
```

Comment:
Can you add a comment with the output of mvn to see the logic of mvn?
As in:
https://github.com/aquasecurity/trivy/blob/20acc231ccd62ccd6cfe413be287f70f2cb12a23/pkg/dependency/parser/java/pom/parse_test.go#L1537-L1539

For `mvn` tp work properly - you need to copy [org](https://github.com/aquasecurity/trivy/tree/20acc231ccd62ccd6cfe413be287f70f2cb12a23/pkg/dependency/parser/java/pom/testdata/repository/org) dir in `.m2` directory.

Reply:
Done